deepchem/molnet/load_function/tox21_datasets.py

```diff
@@ -6,29 +6,16 @@
 from __future__ import unicode_literals
 
 import os
-import deepchem
+import deepchem as dc
 
 
-def load_tox21(featurizer='ECFP', split='index', reload=True, K=4):
+def load_tox21(featurizer='ECFP', split='index'):
   """Load Tox21 datasets. Does not do train/test split"""
   # Featurize Tox21 dataset
   if "DEEPCHEM_DATA_DIR" in os.environ:
     data_dir = os.environ["DEEPCHEM_DATA_DIR"]
   else:
     data_dir = "/tmp"
-  if reload:
```

Comment:
Could you add the `reload` logic back in? It's a nice convenience to ensure that datasets are reloaded if they've already been featurized.